### ISSUE TITLE
Version 1.0.0 is reporting an error state

### ISSUE THREAD

**Issue by leahjlou** [jitpack/jitpack.io#4490]
Version 1.0.0 is reporting an error state.

Link to build log: https://jitpack.io/com/github/ralfstuckert/pdfbox-layout/1.0.0/build.log

In my project, this is the error I'm seeing:

```'
Execution failed for task ':compileJava'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not find com.******.ralfstuckert.pdfbox-layout:pdfbox2-layout:1.0.0.
```

**Comment by fatin-qa** [comment]
I have the same issue. Was working a few days ago. Not sure if the file is removed from maven

`Could not find artifact com.github.ralfstuckert.pdfbox-layout:pdfbox2-layout:jar:1.0.0 in central (https://repo.maven.apache.org/maven2)`

**Comment by Joel-Vitelli** [comment]
Hello. I have the same issue. When i tried to run my project in GitHub Actions i get this error:

```
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  29.893 s
[INFO] Finished at: 2021-03-29T21:01:45Z
[INFO] ------------------------------------------------------------------------
Error:  Failed to execute goal on project Autoc0de-Web: Could not resolve dependencies for project com.autoc0de:Autoc0de-Web:jar:1.0-SNAPSHOT: Could not find artifact com.github.ralfstuckert.pdfbox-layout:pdfbox2-layout:jar:1.0.0 in jitpack.io (https://jitpack.io) -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
Error: Process completed with exit code 1.
```

**Comment by Joel-Vitelli** [comment]
I have a TEMPORARY solution to this problem. Just add these dependencies in our pom.xml

```
    <repositories>
             <repository>
                 <id>jitpack.io</id>
                 <url>https://jitpack.io</url>
             </repository>
    </repositories>
```

```
<!-- https://mvnrepository.com/artifact/com.github.ralfstuckert.pdfbox-layout/pdfbox2-layout -->
        <dependency>
            <groupId>com.github.ralfstuckert.pdfbox-layout</groupId>
            <artifactId>pdfbox2-layout</artifactId>
            <version>0.8.6</version>
        </dependency>

```

This allows us to download the required library, since it seems that the previous repository is failing. However, we are adding an old dependency, so I clarify that it is a temporary solution

**Comment by leahjlou** [comment]
@Joel-Vitelli yes we downgraded to v0.9 to solve our problem temporarily, but would love to see the build fixed.

**Comment by grasshopper7** [comment]
Expanding on the solution provided here by @cdietze - https://github.com/ralfstuckert/pdfbox-layout/issues/88#issuecomment-815727906

The jar is also available in Mulesoft repository (https://repository.mulesoft.org/nexus/content/repositories/public/com/github/ralfstuckert/pdfbox-layout/ and https://mvnrepository.com/artifact/com.github.ralfstuckert.pdfbox-layout/pdfbox2-layout/1.0.0). Just need to add this repository declaration instead of Jitpack.

<repositories>
	<repository>
		<id>mulesoft-releases</id>
		<name>MuleSoft Repository</name>
		<url>https://repository.mulesoft.org/nexus/content/repositories/public/</url>
	</repository>
</repositories>

**Comment by arunqa2016** [comment]
> I have a TEMPORARY solution to this problem. Just add these dependencies in our pom.xml
> 
> ```
>     <repositories>
>              <repository>
>                  <id>jitpack.io</id>
>                  <url>https://jitpack.io</url>
>              </repository>
>     </repositories>
> ```
> 
> ```
> 
>         <dependency>
>             <groupId>com.github.ralfstuckert.pdfbox-layout</groupId>
>             <artifactId>pdfbox2-layout</artifactId>
>             <version>0.8.6</version>
>         </dependency>
> ```
> 
> This allows us to download the required library, since it seems that the previous repository is failing. However, we are adding an old dependency, so I clarify that it is a temporary solution

This is working now for me...thanks for solution.

**Comment by joelCOR** [comment]
Hi!, The issue is back, this time the file is broken from the Mulesoft repo.

Error:  Failed to execute goal on project automation.COR: Could not resolve dependencies for project com.cor.automation:automation.COR:jar:1.0-SNAPSHOT: Could not find artifact com.github.ralfstuckert.pdfbox-layout:pdfbox2-layout:jar:1.0.0 in central (https://repo.maven.apache.org/maven2) -> [Help 1]

**Comment by joelCOR** [comment]
> I have a TEMPORARY solution to this problem. Just add these dependencies in our pom.xml
> 
> ```
>     <repositories>
>              <repository>
>                  <id>jitpack.io</id>
>                  <url>https://jitpack.io</url>
>              </repository>
>     </repositories>
> ```
> 
> ```
> 
>         <dependency>
>             <groupId>com.github.ralfstuckert.pdfbox-layout</groupId>
>             <artifactId>pdfbox2-layout</artifactId>
>             <version>0.8.6</version>
>         </dependency>
> ```
> 
> This allows us to download the required library, since it seems that the previous repository is failing. However, we are adding an old dependency, so I clarify that it is a temporary solution

This first solution its working now.

**Comment by sthepatel** [comment]
It's not working for mulesoft repo and it can be confirmed by going to the [Maven Repo site ](https://mvnrepository.com/artifact/com.github.ralfstuckert.pdfbox-layout/pdfbox2-layout/1.0.0)and clicking the [`jar` link](https://repository.mulesoft.org/nexus/content/repositories/public/com/github/ralfstuckert/pdfbox-layout/pdfbox2-layout/1.0.0/pdfbox2-layout-1.0.0.jar) - normally, we would be able to download the file directly but link doesn't work anymore.

```
Could not find pdfbox2-layout-1.0.0.jar (com.github.ralfstuckert.pdfbox-layout:pdfbox2-layout:1.0.0).
Searched in the following locations:
    https://repository.mulesoft.org/nexus/content/repositories/public/com/github/ralfstuckert/pdfbox-layout/pdfbox2-layout/1.0.0/pdfbox2-layout-1.0.0.jar
```

**Comment by sthepatel** [comment]
### I was able to solve this!

So it looks like `pdfbox2-layout v1.0.0` was related in Sept 2018 on Mulesoft repo - this is the one that is no longer available/working.

However, a new version was released on Jun 1, 2021 — v1.0.1 to the [PentahoOmni Repository](https://mvnrepository.com/repos/pentaho-omni) - [https://mvnrepository.com/artifact/com.github.ralfstuckert.pdfbox-layout/pdfbox2-layout/1.0.1](https://mvnrepository.com/artifact/com.github.ralfstuckert.pdfbox-layout/pdfbox2-layout/1.0.1)

So to use this, just add the PentahoOmni repo to the project - gradle example:

```gradle
repositories {
    mavenCentral()
    maven {
        url 'https://nexus.pentaho.org/content/groups/omni/'
    }
}
```

**Comment by joelCOR** [comment]
I solved the issue updating all dependencies on Pom.xml file and adding Mulesoft repos:

` <repositories>
        <repository>
            <id>mulesoft-releases</id>
            <name>MuleSoft Repository</name>
            <url>https://repository-master.mulesoft.org/releases/</url>
            <layout>default</layout>
        </repository>
        <repository>
            <id>mulesoft-snapshots</id>
            <name>MuleSoft Snapshot Repository</name>
            <url>https://repository-master.mulesoft.org/snapshots/</url>
            <layout>default</layout>
        </repository>
    </repositories>`


and the dependence:

`<dependency>
            <groupId>tech.grasshopper</groupId>
            <artifactId>extentreports-cucumber6-adapter</artifactId>
            <version>2.8.4</version>
        </dependency>`
